catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -111,6 +111,8 @@
 
   protected static final XMLInputFactory XML_INPUT_FACTORY;
 
+  private static int METADATA_MAXIMUM_BYTES;
```

Comment:
`private static final int METADATA_MAXIMUM_BYTES = getMetadataSizeLimit();`  I think would address some of the other comments.

Reply:
Did this and addresses multiple. fixed.